common/djangoapps/student/management/commands/transfer_students.py

```diff
@@ -52,7 +52,6 @@ class Command(TrackedCommand):
                     help="If True, try to transfer certificate items to the new course.")
     )
 
-    @transaction.commit_manually
```

Comment:
Hey - let me on that soapbox! :smile: 

Totally agree that 1) is better code/procedure. Just worried about the "hey - this command changed - rollback the release!" scenario. If we're willing to take on some of that risk (maybe release notes would help?), let's do it.